viewer_clients/cmdline_client/cmd_line_client.py

```diff
@@ -466,51 +486,55 @@ def register_client_command_line(argument_parser):
     subparsers = argument_parser.add_subparsers()
 
     # List runs.
-    listruns_parser = subparsers.add_parser('runs', help='Get the run data.')
+    listruns_parser = subparsers.add_parser('runs',
+                                            formatter_class=ADHFormatter,
+                                            help='Get the run data.')
     listruns_parser.add_argument('--host', type=str, dest="host",
                                  default='localhost',
                                  help='Server host.')
     listruns_parser.add_argument('-p', '--port', type=str, dest="port",
-                                 default=11444,
-                                 required=True, help='HTTP Server port.')
+                                 default="8001", help='HTTP Server port.')
```

Comment:
Is changing the port intentional here and some other places? Why did you make it a string? Maybe factoring these ports out to somewhere is a good idea.